src/main.ts

```diff
@@ -0,0 +1,45 @@
+import { app } from 'electron';
+import { emptyDirSync } from 'fs-extra';
+import path from 'path';
+
+const relaunchApp = () => {
+  app.relaunch();
+  app.exit();
+};
+
+/**
+ * Removes the contents of the user data directory.
+ * This directory is then recreated after re-launching the application.
+ */
+function clearUserDataDirectory(): void {
+  const userDataPath = app.getPath('userData');
+  emptyDirSync(userDataPath);
+  relaunchApp();
+};
+
+/**
+ * Removes directories containing leveldb databases.
+ * Each directory is reinitialized after re-launching the application.
+ */
+function clearLevelDbDirectories(): void {
```

Comment:
```suggestion
function clearSensitiveDirectories(): void {
```

That it's leveldb related files is an implementation detail and not something the caller needs to be aware of.